python-diamondpurvis/hello-diamond.py

```diff
@@ -0,0 +1,19 @@
+#Hard Coded
```

Comment:
Delete this file and leave the hello-DiamondPurvis.py file

Reply:
Deleted file